gammapy/datasets/flux_points.py

```diff
@@ -275,107 +296,102 @@ def plot_residuals(self, ax=None, method="diff", **kwargs):
 
         Parameters
         ----------
-        ax : `~matplotlib.pyplot.Axes`
-            Axes object.
-        method : {"diff", "diff/model", "diff/sqrt(model)"}
-            Method used to compute the residuals, see `MapDataset.residuals()`
+        ax : `~matplotlib.axes.Axes`
+            Axes to plot on.
+        method : {"diff", "diff/model"}
```

Comment:
Why the method "diff/sqrt(model)" disappears in this function? I might have not catched the global logic...

Reply:
Because an error is raised on line 312/331 for that `yerr` option.